Sources/Extensions/UIKit/UIView+Povio.swift

```diff
@@ -55,12 +55,12 @@ public extension UIView {
     static let shadowOpacity = "shadowOpacityKey"
   }
   
-  func rotate() {
+  func rotate(speed: CFTimeInterval = 1.25, clockWiseDirection: Bool = true) {
```

Comment:
I think `clockWiseDirection` can be shortened to just `clockwise'.

Reply:
Good one 👍

Reply:
@tonikocjan thanks for the suggestion, we can remove 'direction' since it can be implied from the name of the function.